### ISSUE TITLE
Restore "Sticky Issues Header" feature

### ISSUE THREAD

**Issue by fregante** [sindresorhus/refined-github#2280]
It was added in https://github.com/sindresorhus/refined-github/pull/1311 but I can't find it anymore.

Anyone remember if it was removed for any reason? cc @tommoor @hkdobrev @yakov116 

Anyway this is a quick fix if anyone wants to pick it up.

**Comment by sharkykh** [comment]
```
$ git log -S'/* Sticky table header on issues list */' --pretty=format:'%h %an -- %s -- %ad'
a9ec680 Waseem Dahman -- Fix sticky file header tooltips (#1581) -- 2018-11-06 17:43:48 +0200
dd8760b Tom Moor -- 📌Sticky Issues Header (#1311) -- 2018-05-10 20:13:57 +0300
```
a9ec6800cff605ca6c314b6ea427fccf5795b6e5 combined this and the sticky file headers features together, and changed the comment from `/* Sticky table header on issues list */` to `/* Sticky file and table headers */`
```
$ git log -S'/* Sticky file and table headers */' --pretty=format:'%h %an -- %s -- %ad'
76cc04d Federico Brigante -- Drop feature `make-headers-sticky` -- 2019-03-18 19:18:55 +0200
a9ec680 Waseem Dahman -- Fix sticky file header tooltips (#1581) -- 2018-11-06 17:43:48 +0200
```
76cc04d4b69974ce4aa8c02ed2afedb66ad66cd8 removed both sticky issue list headers and sticky file headers (the latter were implemented by GitHub).

To whoever picks this up,
keep in mind that dd8760bf324c665088790df8be37a647882a0d88 (#1311) also had a bugfix commit later: 4f43ef673281ce0dec28a88d7893043fcf1a6ca2 (#1318)
And maybe it should have an option to disable - so, have a `.tsx` file to inject a `rgb-sticky-header` class, like https://github.com/sindresorhus/refined-github/commit/a9ec6800cff605ca6c314b6ea427fccf5795b6e5#diff-b4d4b46a0592d8243f214367bdfcf5bc added.

**Comment by fregante** [comment]
Excellent! Thank you so much for that!
